pwa/src/templates/templateParts/dependenciesTemplates/relationsDependenciesTemplate/RelationsDependenciesTemplate.tsx

```diff
@@ -6,10 +6,10 @@ import _ from "lodash";
 import { getTokenValue } from "../../../../services/getTokenValue";
 import { addNewLineToString } from "../../../../services/addNewLineToString";
 import { navigate } from "gatsby";
-import { NodeToolTip } from "../../../../components/toolTip/ToolTip";
 import { NotificationPopUp as _NotificationPopUp } from "@conduction/components";
 import "vis-network/styles/vis-network.css";
 import { renderToStaticMarkup } from "react-dom/server";
+import { NodeToolTip } from "@conduction/components/lib/components/toolTip/ToolTip";
```

Comment:
This is causing your build to break. See screenshot.

![image](https://user-images.githubusercontent.com/66552932/193521616-24103bd7-168b-488f-8efc-92ce396e3097.png)